build/terraform/gke/module.tf

```diff
@@ -141,3 +141,22 @@ output "token" {
 output "cluster_ca_certificate" {
   value = module.gke_cluster.cluster_ca_certificate
 }
+
+// Additional resources for terratests
+
+// Template used in "terraform-google-modules/kubernetes-engine/google//modules/auth"
+
+data "template_file" "kubeconfig" {
```

Comment:
Should this be part of the main template? Or split out somewhere?

Since this isn't part of a production deploy, it's probably would be better to be it's own module ?

Reply:
That's right, let me update this.

Reply:
We have a separate `install/terraform/modules/gke` which is used in examples, this one is build pipeline only. But let me check if there is a way to split this thing with kubeconfig generation.